data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg

```diff
@@ -139,7 +139,7 @@
 
         [message]
             speaker=Pruol
-            message= _ "Argh! They are stronger than we thought. Someone go tell the Master..."
+            message= _ "Argh! They are stronger than we thought. Someone, go tell the Master..."
```

Comment:
If it was a proper noun then I'd agree with the comma, but for "someone" I don't think it fits.

Reply:
Right.